api/Document.json

```diff
@@ -11049,7 +11049,8 @@
               "version_added": "51"
             },
             "ie": {
-              "version_added": "10"
+              "version_added": "10",
+              "notes": "This event's handler attribute (<code>ontransitionend</code>) is not supported; however, the event itself is supported since IE 10. The event can be listened to via <code>document.addEventListener('transitionend', function() {});</code>."
```

Comment:
Can you match the wording of the visibilitychange_event entry, or change it to match this if you think it's better like this?